app/main.py

```diff
@@ -311,10 +326,10 @@ def display_alert_frame_metadata(n_clicks_marker, img_url):
 
 @app.callback(
     Output('interval-component', 'disabled'),
-    [Input("display_alert_frame_btn{}".format(alert_id), 'n_clicks')])
+    [Input("alert_marker_{}".format(alert_id), 'n_clicks')])
```

Comment:
So you're coming down to the `alert_marker` level because not possible to combine `"display_alert_frame_btn{}".format(alert_id), 'n_clicks'` and `'acknowledge_alert_checkbox_{}'.format(alert_id), 'checked'` as inputs as they don't occur as the same time right ?

Reply:
To be fully transparent, I came back to the marker level because this was the quickest fix I saw! 

But to answer your remark, for the reason you mention, we could not combine both inputs indeed. Maybe there is a syntax to have the callback triggered by one input **or** the other. Or maybe we could write the same logic with a `State`. 

But since this callback is not supposed to be kept anyway, I have not spent much time looking for other options at hand.